src/test/java/com/game/Battleship/controllers/BoardControllerTests.java

```diff
@@ -18,27 +23,31 @@
     private TestRestTemplate template;
 
     @Test
-    public void boardEndpointContainsSizeAttribute(){
+    public void boardsEndpointContains2Boards() {
 
-        Board response = template.getForObject("/board", Board.class);
-        Assertions.assertEquals(10, response.getSize());
+        List<Object> response = template.getForObject("/boards", List.class);
+        Assertions.assertEquals(2, response.size());
 
     }
 
     @Test
-    public void boardFieldSizeIsEqualWithBoardSizeToThePowerOf2(){
+    public void boardsEndpointHasABoardObject() {
 
-        Board response = template.getForObject("/board", Board.class);
-        Assertions.assertEquals(100, response.getBoardField().size());
+        ResponseEntity<Board[]> response = template.getForEntity("/boards", Board[].class);
+        Assert.isInstanceOf(Board.class, Arrays.stream(response.getBody()).findFirst().get());
 
     }
 
     @Test
-    public void boardFieldContainsCellObjects(){
+    public void boardsEndpointNameGetRequestReturnStatusOK() {
 
-        Board response = template.getForObject("/board", Board.class);
-        Assert.isInstanceOf(Cell.class, response.getBoardField().get(0));
+        ResponseEntity<String> response = template.getForEntity("/boards", String.class);
+        Assertions.assertEquals(response.getStatusCode(), HttpStatus.OK);
 
     }
 
 }
+
+
+
+
```

Comment:
Only need one empty line at the end of a file

Reply:
Done